frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -7,7 +7,7 @@
     :a11yTitle='title'
   )
     template(slot='subtitle')
-      i18n(v-if='groupShouldPropose') New proposal
+      i18n(v-if='shouldPropose') New proposal
```

Comment:
Not clear to me that these changes are needed either

Reply:
Maybe not needed, but doesn't seem to be harmful, and shorter so why not